spec/models/item_category_spec.rb

```diff
@@ -0,0 +1,3 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.